src/display/text_layer.js

```diff
@@ -142,12 +138,8 @@ const renderTextLayer = (function renderTextLayerClosure() {
       scale: 1,
     };
 
+    textDiv.textContent = geom.str;
```

Comment:
This looks like a mistake, since there's already an *identical* line below; please remove this.